### PR TITLE
[GEOS-10255] i18n use classes, selfclear and leftwise for layout

### PR DESCRIPTION
**Pull request by jodygarnett**
[GEOS-10255](https://osgeo-org.atlassian.net/browse/GEOS-10255)

This code provides some fixes to the new i18n fragments allowing layout using csss (rather than hard coded styles and br).

Signed-off-by: Jody Garnett <jody.garnett@gmail.com>

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
- [x] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
- [x] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [x] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).

For core and extension modules:

- [ ] New unit tests have been added covering the changes.
- [ ] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en/user/source) has been updated (if change is visible to end users).
- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
- [x] There is an issue in the https://osgeo-org.atlassian.net/browse/GEOS-10255
- [x] Commit message(s) must be in the form ``[GEOS-XYZWV] Title of the Jira ticket``.
- [x] Bug fixes and small new features are presented as a single commit.
- [ ] Each commit has a single objective (if there are multiple commits, each has a separate JIRA ticket describing its goal).

### CODE REVIEW

**Review by taba90 — APPROVED** [review]
looks good to me.

File: src/web/core/src/main/java/org/geoserver/web/css/geoserver.css [hunk 1]
```diff
@@ -925,6 +925,33 @@ padding-top: 18px;
...
+}
+
+.i18n-tile-and-abstract .text {
+  width: 100%;
+}
+
+.i18n-panel .text {
+  width: 100%;
+}
+*/
```
Comment (jodygarnett) [inline src/web/core/src/main/java/org/geoserver/web/css/geoserver.css:954]: The above is commented out, could not manage to get the i18n table to have a sensible layout.

Comment (aaime) [reply src/web/core/src/main/java/org/geoserver/web/css/geoserver.css:954]: Commented out bits are not welcomed in PRs though?
Unfortunately we don't have any CSS expert in our server-side team...

**Comment by jodygarnett** [comment]
Thanks for testing @taba90, I removed the commended out code @aaime.

With feedback addressed I will merge and backport.

**Pull request merged**